string control_topic
string mqtt_reference
float64 frequency
bool latched
int8 qos